easybuild/easyconfigs/n/NVHPC/NVHPC-21.5.eb

```diff
@@ -0,0 +1,77 @@
+##
+# Author:    Robert Mijakovic <robert.mijakovic@lxp.lu>
+##
+name = 'NVHPC'
+version = '21.5'
+
+homepage = 'https://developer.nvidia.com/hpc-sdk/'
+description = """C, C++ and Fortran compilers included with the NVIDIA HPC SDK (previously: PGI)"""
+
+toolchain = SYSTEM
+
+# By downloading, you accept the HPC SDK Software License Agreement (https://docs.nvidia.com/hpc-sdk/eula/index.html)
+accept_eula = True
+source_urls = ['https://developer.download.nvidia.com/hpc-sdk/%(version)s/']
+local_tarball_tmpl = 'nvhpc_2021_%%(version_major)s%%(version_minor)s_Linux_%s_cuda_multi.tar.gz'
+sources = [local_tarball_tmpl % '%(arch)s']
+checksums = [
+    {
+        local_tarball_tmpl % 'aarch64':
+            '1a1748cd7cf538199d92ab3b1208935fa4a62708ba21125aeadb328ddc7380d4',
+        local_tarball_tmpl % 'ppc64le':
+            '4674931a5ce28724308cb9cebd546eefa3f0646d3d08adbea28ba5ad27f0c163',
+        local_tarball_tmpl % 'x86_64':
+            '21989e52c58a6914743631c8200de1fec7e10b3449c6c1833f3032ee74b85f8e',
+    }
+]
+
+local_gccver = '10.3.0'
+dependencies = [
+    ('GCCcore', local_gccver),
+    ('binutils', '2.36.1', '', ('GCCcore', local_gccver)),
+    # This is necessary to avoid cases where just libnuma.so.1 is present in the system and -lnuma fails
+    ('numactl', '2.0.14', '', ('GCCcore', local_gccver))
+]
+
+# specify default CUDA version that should be used by NVHPC
+# should match one of the CUDA versions that are included with this NVHPC version
+# (see install_components/Linux_x86_64/21.5/cuda/)
+# for NVHPC 21.5, those are: 11.2, 11.0, 10.2;
```

Comment:
@robert-mijakovic CUDA versions in this comment need to be fixed, since 11.3 isn't mentioned?